keps/sig-auth/1687-hierarchical-namespaces-subproject/README.md

```diff
@@ -0,0 +1,192 @@
+  - [Hierarchical Namespace Controller Continues As A Non-Sig Project](#hierarchical-namespace-controller-continues-as-a-non-sig-project)
+  - [Abandon Hierarchical Namespace Controller](#abandon-hierarchical-namespace-controller)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+
+## Summary
+
+[Hierarchical Namespace Controller](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/hnc)
+makes it easier for you to create and manage namespaces in your cluster.
+For example, you can create a hierarchical namespace under your team's namespace,
+even if you don't have cluster-level permission to create namespaces, and easily
+apply policies like RBAC and Network Policies across all namespaces in your
+team (e.g. a set of related microservices).
+
+You can read more about hierarchical namespaces in the
+[HNC Concepts doc](https://docs.google.com/document/d/1R4rwTweYBWYDTC9UC-qThaMkPk8gtflr_ycHOOqzBQE/edit).
+
+
+## Motivation
+
+The Hierarchical Namespace Controller is currently being developed within the
+Multi-Tenancy Working Group, (of which Sig-Auth is the sponsor). As Working Groups
+are not meant to own code, and Hierarchical Namespace Controller is nearly to
+MVP status, a permant home is required. Additionally, having a permanent home
+for Hierarchical Namespace Controller prior to officially releasing will prevent
+cumbersome migrations of client libraries if a move were to happen at a later time.
+
+
+## Goals
+
+Establish a new repository and permanent home for Hierarchical Namespace
+Controller at github.com/kubernetes-sigs/hierarchical-namespaces to be
+maintained by the open source Kubernetes community and governed as a subproject
+of sig-auth.
+
+
+## Proposal
+
+The current multi-tenancy repository will be transferred directly. Subsequent
+pull requests will then remove HNC from the multi-tenancy repository and
+reorganize the new Hierarchical Namespace Controller repository as needed.
+The new source control location will become the authoritative source of truth
+for all issues and pull requests. As an independent subproject under sig-auth,
+Hierarchical Namespace Controller will continue to maintain the Apache license
+hosted [here](https://github.com/kubernetes-sigs).
+
+The following group of community members will serve as initial maintainers of
+the new repository:
+
+* @adrianludwin
+* @rjbez17
+
+Maintainers will devote time to transfer, maintain, and release the Hierarchical
+Namespace Controller code base in a time bound manor. Maintainers will document
+features, blog, evangelize, and respond to the community on slack, groups,
+forums, etc. Maintainers will serve as the initial owners of the subproject.
+
+
+## Risks and Mitigations
+
+ There are no obvious risks with this proposal. Hierarchical Namespace
+ Controller is currently in pre-alpha and has no apparent adoption.
+
+
+## Graduation Criteria
```

Comment:
Do these need to happen before or after the repo move?

Reply:
What - the graduation criteria? I think that's up to you. We're ok just moving it first and doing any necessary reviews, upgrades etc later - but sig-auth may have more exacting standards :)

Reply:
I'm wondering what you want "graduation" to mean here and the ordering of events you are thinking about. What is forcing completion of the work listed here? What changes when the work is complete?

Reply:
Updated based on feedback given offline